lib/modules/manager/terraform/lockfile/hash.ts

```diff
@@ -86,7 +86,7 @@ export class TerraformProviderHash {
       return hash;
     } finally {
       // delete zip file
-      await fs.unlink(downloadFileName);
+      await fs.rmCache(downloadFileName);
```

Comment:
I think we can safely use `rm` instead of `unlink` in order to remove zip file

Reply:
don't know the difference between those functions 🤷‍♂️